etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,448 @@
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.60"
+    port: 9292
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 9292
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 9101
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'haproxy_exporte' in docker container 'prometheus_haproxy_exporter'" 
+  - interface: "all"
+    port: 111
+    proto: tcp
+    comment: "hint: used by 'rpcbind'" 
+  - interface: "{{ internal_interface }}"
+    port: 80
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'httpd' in docker container 'horizon'" 
+  - interface: "all"
+    port: 8080
+    proto: tcp
+    comment: "hint: used by 'nginx' in docker container 'inspection_store'" 
+  - interface: "{{ internal_interface }}"
+    port: 9104
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'mysqld_exporter' in docker container 'prometheus_mysqld_exporter'" 
+  - interface: "lo"
+    port: 6640
+    proto: tcp
+    destination: 127.0.0.1
+    comment: "hint: used by 'ovsdb-server' in docker container 'openvswitch_db'" 
+  - interface: "breno1.65"
+    port: 9200
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.60"
+    port: 80
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 80
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 6385
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'ironic-api' in docker container 'ironic_api'" 
+  - interface: "all"
+    port: 4369
+    proto: tcp
+    comment: "hint: used by 'epmd' in docker container 'rabbitmq'" 
+  - interface: "breno1.60"
+    port: 6385
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 6385
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 8786
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'manila-api' in docker container 'manila_api'" 
+  - interface: "breno1.60"
+    port: 8786
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 8786
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "all"
```

Comment:
Perhaps we should lock this down to just the admin interface?

Reply:
Because we haven't switched the controller to the admin network. Kayobe can no longer connect after you lock this down to devops:

```
fatal: [svn1-dr06-u8]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 10.61.102.0 port 22: Connection refused\r\n", "unreachable": true}
```